.pre-commit-config.yaml

```diff
@@ -80,7 +80,7 @@ repos:
         - --docstring-convention=numpy
         - --max-line-length=120
         - --extend-immutable-calls=Query,fastapi.Depends,fastapi.params.Depends
-        - --ignore=B008 # Ignore error for function calls in argument defaults
+        - --ignore=B008,W503 # Ignore error for function calls in argument defaults
```

Comment:
I think we should probably keep W503 to keep with the pep standards https://www.flake8rules.com/rules/W503.html

where did this happen?

Reply:
happened in create model state machine implementation. One of our formatters wants to put each condition on its own line, and flake8 doesn't. Couldn't make them both happy at the same time.
`lambda/models/state-machine/create-model.py:38`

Reply:
Oh for that we could try putting the `and`'s at the end of the line.  i don't think there's anything wrong with needing the backslash at the end to complete the line either. otherwise we can just do an inline ignore so we ignore the one line, but not the rest. so adding a comment like `# noqa: W503` on the specific line could work